blueprints/soci-snapshotter/README.md

```diff
@@ -150,37 +182,81 @@ spec:
     --//
 ```
 
+### Bottlerocket
+
+SOCI is integrated into Bottlerocket latest AMIs and simplify setup and configuration through Bottlerocket APIs as in the example below.
+
+In Bottlerocket, SOCI's data dir is configured at `/var/lib/soci-snapshotter`, to take advantage of instances with NVMe disks, we will need to configure ephemeral storage through Bottlerocket's Settings API, replacing `instanceStorePolicy: RAID0` with `[settings.bootstrap-commands.k8s-ephemeral-storage]` as you can see below, we added `/var/lib/soci-snapshotter` as a bind dir.
+
+```yaml
+apiVersion: karpenter.k8s.aws/v1
+kind: EC2NodeClass
+metadata:
+  name: soci-snapshotter-br
+...
+...
+spec:
+...
+...
+  userData: |
+    [settings.container-runtime]
+    snapshotter = "soci"
+    [settings.container-runtime-plugins.soci-snapshotter]
+    pull-mode = "parallel-pull-unpack"
+    [settings.container-runtime-plugins.soci-snapshotter.parallel-pull-unpack]
+    max-concurrent-downloads-per-image = 10
+    concurrent-download-chunk-size = "16mb"
+    max-concurrent-unpacks-per-image = 10
+    discard-unpacked-layers = true
+    [settings.bootstrap-commands.k8s-ephemeral-storage]
+    commands = [
+        ["apiclient", "ephemeral-storage", "init"],
+        ["apiclient", "ephemeral-storage" ,"bind", "--dirs", "/var/lib/containerd", "/var/lib/kubelet", "/var/log/pods", "/var/lib/soci-snapshotter"]
+    ]
+    essential = true
+    mode = "always"
+```
+
 ## Results
 
 Wait until the pods from the sample workload are in running status:
 ```sh
 > kubectl wait --for=condition=Ready pods --all --namespace default --timeout=300s
-pod/vllm-85b4fbdd49-k88gg condition met
-pod/vllm-soci-744dfdbcfb-d4bwj condition met
+pod/vllm-59bfb6f86c-9nfxb condition met
+pod/vllm-soci-6d9bfd996d-vhr4j condition met
+pod/vllm-soci-br-74b59cc4bd-rq8cw condition met
 ```
 
 The sample workload deploys two Deployments running [Amazon Deep Learning Containers for vLLM](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/dlc-vllm-x86-ec2.html), one using SOCI parallel pull/unpack mode and one remains using the default containerd implementation.
 
 Let's examine the pull time for each Deployment:
 
-The `vllm` deployment using the default containerd implementation results in pull time of **2m18.049s**.
+The `vllm` deployment using the default containerd implementation results in pull time of **1m52.33s**.
 ```sh
-> kubectl describe pod/vllm-85b4fbdd49-k88gg | grep Pulled
-  Normal   Pulled            3m43s  kubelet            Successfully pulled image "763104351884.dkr.ecr.us-east-1.amazonaws.com/vllm:0.9-gpu-py312-ec2" in 2m18.049s (2m18.049s including waiting). Image size: 10757041023 bytes.
+> kubectl describe pod/vllm-59bfb6f86c-9nfxb | grep Pulled
```

Comment:
I didn't think about this, but why don't you simplify the command using something like this?

`kubectl describe pods -l app=vllm | grep Pulled`

That way people don't have to figure out the pod name

Reply:
I noticed something, I was distracted and I read your comment saying that the pull time was 1m and I saw 7ms in the output, it was until I scrolled to the right when I saw the correct time you mentioned, why don't we wrap the output? like this:

```
Normal   Pulled            3m43s  kubelet            
  Successfully pulled image "763104351884.dkr.ecr.us-east-1.amazonaws.com/vllm:0.9-gpu-py312-ec2" 
  in 2m18.049s (2m18.049s including waiting). Image size: 10757041023 bytes.
```